modules/platforms/dotnet/Apache.Ignite.Core/Cache/Query/IQueryCursorFieldMetadata.cs

```diff
@@ -0,0 +1,43 @@
+﻿/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Apache.Ignite.Core.Cache.Query
+{
+    using System;
+
+    /// <summary>
+    /// Query field descriptor. This descriptor is used to provide metadata
+    /// about fields returned in query result.
+    /// </summary>
+    public interface IQueryCursorFieldMetadata
+    {
+        /// <summary>
+        /// Field name
```

Comment:
`Gets the field name.`
https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1623.md